module/mempool/stdmap/backdata/mapBackData.go

```diff
@@ -82,6 +82,26 @@ func (b MapBackData) All() map[flow.Identifier]flow.Entity {
 	return entities
 }
 
+func (b MapBackData) Identifiers() flow.IdentifierList {
+	ids := make(flow.IdentifierList, len(b.entities))
+	i := 0
+	for entityID := range b.entities {
```

Comment:
maybe replace 'range' with 'for' loop, or maybe use the 'for i, entityID' syntax.

Reply:
`b.entities` is a map and we may not be able to iterate on it using a for loop unless through reflection. Not sure if it is the desired case?

Reply:
looks like Go prevents getting access to a pure key collection.  You're right.